app/models/etl/dirty.rb

```diff
@@ -0,0 +1,30 @@
+class ETL::Dirty
```

Comment:
What about renaming this class to: `ETL::OutdatedItems`? or something more explicit about the fact that we are promoting new versions of the items?